### DISCUSSION TITLE
Confirm channel choosen

### DISCUSSION THREAD

**Discussion by DinhHuy2010** [ThioJoe/YT-Spammer-Purge#658] [Ideas]
Could you add feature to confirm channel selected? I see this discussion from https://github.com/ThioJoe/YT-Spammer-Purge/discussions/614 and https://github.com/ThioJoe/YT-Spammer-Purge/issues/455, https://github.com/ThioJoe/YT-Spammer-Purge/issues/624 (link to issues)

And I want to get all channel id from the given channel link to choose what the the
channel you want to get and confirm the selection. (Use the YouTube API)

**Comment by Firecul** [comment]
> Could you add feature to confirm channel selected?

You can always type `X` in the prompt to go back to the main menu if it's wrong.

![WindowsTerminal_Windows_PowerShell (31)](https://user-images.githubusercontent.com/34950371/154097790-458e6d97-df45-4644-af30-529ff717e767.png)

> And I want to get all channel id from the given channel link to choose what channel you want to get

There should only ever be a single ID from a channel link afaik.

---

I'm thinking we might not be understanding your full meaning/questions from previous discussions tbh, I certainly am not.

**Comment by DinhHuy2010** [comment]
What about get all channel id from channel link to choose from?
